### PR TITLE
sm2 config

### PR DESCRIPTION
**Pull request by zhihuixinyuan**
添加SM2算法

### CODE REVIEW

File: packages/cita-signer/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cryptape/cita-signer",
-  "version": "2.5.2",
+  "version": "2.4.1",
```
Comment (leeyr338) [inline packages/cita-signer/package.json]: why we need downgrade the version?

File: packages/cita-signer/src/index.ts [hunk 2]
```diff
@@ -172,33 +176,83 @@ const signer = (
   tx.setVersion(+version)
 
   const txMsg = tx.serializeBinary()
+  // console.log(txMsg)
```
Comment (leeyr338) [inline packages/cita-signer/src/index.ts]: Delete this comment.

File: packages/cita-signer/package.json (hunk 1 above)
Comment (zhihuixinyuan) [reply packages/cita-signer/package.json]: 不用，新版我测试也通过了

**Comment by rainchen** [comment]
@zhihuixinyuan why closed this PR?

**Pull request closed without merging**